Automattic/jetpack
# Docker: Try updating base image to php apache and allowing multiple PHP versions

#### Changes proposed in this Pull Request:

* Sets base image (`FROM`) to depend on the `php` official image and an ENVIRONMENT variable `DOCKER_PHP_VERSION`.
* Introduces a `.env ` file in the root directory defining `DOCKER_PHP_VERSION` to be `7.0:apache`.
* Introduces the npm commans `yarn docker:up:php7.0`, `yarn docker:up:php7.1` and `yarn docker:up:php7.2`

#### Testing instructions:



* Check this branch
* Run `yarn docker:up`. Expect your WordPress env to be running on PHP 7.2
  * Stop with `yarn docker:stop`. 
* Run `yarn docker:up:php7.0`. Expect your WordPress env to be running on PHP 7.0
  * Stop with `yarn docker:stop`. 
* Run `yarn docker:up:php7.2`. Expect your WordPress env to be running on PHP 7.2
  * Stop with `yarn docker:stop`. 
* Run `yarn docker:up:php7.1`. Expect your WordPress env to be running on PHP 7.1
  * Stop with `yarn docker:stop`. 



#### Proposed changelog entry for your changes:
